client/my-sites/patterns/components/pattern-library/style.scss

```
@@ -4,6 +4,9 @@
 
 .pattern-library__pill-navigation {
 	background: var(--studio-white);
+	position: sticky;
+	top: 0;
```

```suggestion
	top: var(--masterbar-height, 0);
```
Optional, but this CSS var holds the height of the masterbar, so if we use it, we don't need the `.is-logged-in` selector.

---

It's not conditional variable, so it definitely makes sens to add it, but added to https://github.com/Automattic/wp-calypso/pull/88801/commits/778387195d717c349f2be90526faf19601e704a3#diff-f625cff80c23f1d34d059b70ab0cf5431cf4168a471f24c741217b2cb7693f29R33

---

> It's not conditional variable

What do you mean by "conditional variable"?